client/src/components/Projects/ColumnHeaderPopups/ProjectTableColumnHeader.js

```diff
@@ -1,27 +1,54 @@
 import React from "react";
 import PropTypes from "prop-types";
 import "react-datepicker/dist/react-datepicker.css";
-import { MdFilterAlt } from "react-icons/md";
+import { MdFilterList } from "react-icons/md";
```

Comment:
You probably got the idea for this icon from the prototype, but, the prototype is several months old, and the design team has changed the icons they want to use in the design guide See https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit#slide=id.g2e77662f2ce_0_10  page 17.  The upshot is that you need to change the icon back to MdFilterAlt.